### PR TITLE
W-11351864: File connector out of memory error workaround

### PR DESCRIPTION
**Pull request by giss-ignacio**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: file/1.4/modules/ROOT/pages/file-troubleshooting.adoc [hunk 1]
```diff
@@ -33,6 +33,17 @@ To resolve this error:
 
 For additional information about enabling verbose logging for SDK components in your Mule application, see the xref:mule-sdk::troubleshooting.adoc[Troubleshooting SDK components] documentation.
 
+== Troubleshooting Out Of Memory with Repeatable File Store Iterable
+
+In some cases where the file folder contains large files, and you set the *Streaming strategy* field to the *Repeatable file store iterable (Default)* option for the *List* operation, it can cause the Mule application to throw a `java.lang.OutOfMemoryError` message.
```
Comment (sathya0) [inline file/1.4/modules/ROOT/pages/file-troubleshooting.adoc]:
```suggestion
In some cases where the file folder contains large files and you set the *Streaming strategy* field to the *Repeatable file store iterable (Default)* option for the *List* operation, the Mule application might throw a `java.lang.OutOfMemoryError` message.
```

File: file/1.4/modules/ROOT/pages/file-troubleshooting.adoc [hunk 2]
```diff
@@ -33,6 +33,17 @@ To resolve this error:
...
+
+To avoid this issue, set the *Streaming strategy* field from *Repeatable file store iterable (Default)* to the *No repeatable iterable* option.
+
+For additional information about repeatable file store, refer to the xref:mule-runtime::tuning-repeatable-nonrepeatable.adoc#file-store-repeatable-stream[Repeatable File Store] documentation.
```
Comment (sathya0) [inline file/1.4/modules/ROOT/pages/file-troubleshooting.adoc]:
```suggestion
For additional information about repeatable file store, refer to xref:mule-runtime::tuning-repeatable-nonrepeatable.adoc#file-store-repeatable-stream[Repeatable File Store].
```

File: file/1.4/modules/ROOT/pages/file-troubleshooting.adoc [hunk 3]
```diff
@@ -33,6 +33,17 @@ To resolve this error:
...
+
+For additional information about disabling repeatable streaming, refer to the xref:mule-runtime::streaming-about.adoc#disable_repeatable_streaming[Disable repeatable streaming] documentation.
```
Comment (sathya0) [inline file/1.4/modules/ROOT/pages/file-troubleshooting.adoc]:
```suggestion
For additional information about disabling repeatable streaming, refer to xref:mule-runtime::streaming-about.adoc#disable_repeatable_streaming[Disable repeatable streaming].
```

**Review by sathya0 — DISMISSED** [review]

**Comment by kevintroller** [comment]
Closing as recreating new PR https://github.com/mulesoft/docs-connectors/pull/2231

**Pull request closed without merging**